### ISSUE TITLE
Window size doesn't match the package.json

### ISSUE THREAD

**Issue by sz-piotr** [nwjs/nw.js#5093]
As in the title.

**NW.js version**: 0.15.4
**OS**: Windows 10 64 bit

package.json
```json
{
  "name": "App",
  "main": "index.html",
  "window": {
    "width": 800,
    "height": 600
  }
}
```

index.html
```html
<!DOCTYPE html>
<html>
<head>
    <title>App</title>
    <meta charset="UTF-8">
    <style>
        html,
        body {
            width: 100%;
            height: 100%;
            margin: 0;
            padding: 0;
            background-color: blue;
        }
        #yellow {
            color: black;
            background-color: yellow;
            width: 800px;
            height: 600px;
        }
    </style>
</head>
<body>
    <div id="yellow"></div>
</body>
<script>
    window.onload = function() {
        var yellow = document.getElementById('yellow');
        yellow.innerHTML = window.innerWidth + ", " + window.innerHeight;
    }
</script>
</html>
```

I would expect to see the entire window yellow with the text `800, 600` in the corner. Instead it has a blue outline and displays `816, 608`.

If i set the `"frame"` property of the window to `false` however, the result is correct.

![Screenshot](http://i.imgur.com/nBhIvod.png)

**Comment by AshleyScirra** [comment]
This also appears to affect Construct 2 games: https://www.scirra.com/forum/black-borders-in-nw-js_t178871

**Comment by rogerwang** [comment]
The latest nightly build works for me: http://dl.nwjs.io/live-build/07-12-2016/nw16-e410c99-10b4d75-cd7731c-3151ede/v0.16.0/

btw, 0.16.0 will be released soon and replace 0.15.x in the stable branch.
